packages/core/src/App/Containers/RealityCheckModal/reality-check-modal.jsx

```diff
@@ -60,24 +89,59 @@ const RealityCheckModal = ({
         setVisibilityRealityCheck(0);
     };
 
+    const enableMax30dayTurnover = !self_exclusion.max_30day_turnover;
```

Comment:
Can we update this variable name to `snake_case`?

Reply:
Also, booleans should start with is_, has_